catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/SpriteListFragment.java

```diff
@@ -136,7 +137,7 @@ public void onAdapterReady() {
 	}
 
 	@Override
-	public void onPrepareOptionsMenu(Menu menu) {
+	public void onPrepareOptionsMenu(@NotNull Menu menu) {
```

Comment:
This will not change the bahaviour of the function as @NotNull annotation only tells the caller to pass only a parameter that is not null. The caller however is not forced to behave like this and can still pass a parameter that is not null. So this will not solve the NullPointerException that is caused by the parameter

Reply:
Oh I had made one more change, which I forgot to commit 😅